yt/data_objects/construction_data_containers.py

```diff
@@ -2481,13 +2488,35 @@ def export_ply(
             elif sample_type == "vertex" and color_field not in self.vertex_samples:
                 self.get_data(color_field, sample_type, no_ghost=no_ghost)
         self._export_ply(
-            filename, bounds, color_field, color_map, color_log, sample_type
+            filename,
+            bounds,
+            color_field,
+            color_map,
+            color_log,
+            sample_type,
+            color_field_max=color_field_max,
+            color_field_min=color_field_min
         )
 
-    def _color_samples(self, cs, color_log, color_map, arr):
+    def _color_samples(self, cs, color_log, color_map, color_field_max, color_field_min, arr):
         if color_log:
             cs = np.log10(cs)
-        mi, ma = cs.min(), cs.max()
+        if color_field_min is None:
+            cs = [float(field) for field in cs]
+            cs = np.array(cs)
+            mi = cs.min()
+        else:
+            mi = color_field_min
+            if color_log:
+                mi = np.log10(mi)
```

Comment:
shouldn't this be applied also in the case `color_field_min is None` ?

Reply:
no, since the field itself is logged above, then the min is the logged already

Reply:
fair enough, I missed that !